libs/src/api/track.js

```diff
@@ -361,12 +376,27 @@ class Track extends Base {
 
       phase = phases.ASSOCIATING_TRACK
       // Associate the track id with the file metadata and block number
-      await this.creatorNode.associateTrack(
-        trackId,
-        metadataFileUUID,
-        txReceipt.blockNumber,
-        transcodedTrackUUID
-      )
+      await retry(async (bail, num) => {
+        return this.creatorNode.associateTrack(
```

Comment:
since this calls _makeRequest under the hood and that has the retry, do we need the retry here?

Reply:
yeah, i added this one first. happy to remove 👍